static/js/components/Toolbar.js

```diff
@@ -36,7 +39,9 @@ export default class Toolbar extends React.Component<*, void> {
               <a
                 href="#"
                 className="material-icons mdc-toolbar__icon--menu"
-                onClick={toggleShowSidebar}
+                onClick={() =>
+                  toggleShowDrawer(getViewportWidth() < DRAWER_BREAKPOINT)
```

Comment:
Might be missing something, but I think it makes more sense to let `toggleShowDrawer` handle the width determination if no value is passed in for the `mobile` argument. If another component had a need to toggle the drawer, it would have to perform this same logic.

Reply:
yeah, makes sense. I'll move that into the function